workspaces/liferay-sample-workspace/client-extensions/liferay-sample-etc-node/package.json

```diff
@@ -0,0 +1,18 @@
+{
+	"dependencies": {
+		"body-parser": "1.20.2",
+		"cors": "2.8.5",
+		"express": "4.18.2",
+		"jsonwebtoken": "9.0.0",
+		"jwk-to-pem": "2.0.5",
+		"node-fetch": "2.6.9",
+		"winston": "3.8.2"
```

Comment:
Are these all absolutely necessary? Just thinking "how can we simplify as much as possible" so that someone reading this sample doesn't think "oh `jsonwebtoken` is absolutely necessary for this sample

Reply:
I think you removed the use of winston, so you can remove the dependency here